middle_end/flambda/simplify/simplify_set_of_closures.rec.ml

```diff
@@ -408,7 +408,7 @@ let simplify_function context r closure_id function_decl
       (* We need to use [dacc_after_body] to ensure that all [code_ids] in
          [function_decl] are available for the inlining decision code. *)
```

Comment:
I think this should say "[code_ids] in [params_and_body]".

Reply:
I've pushed a fix.